docs/contributing/ASPNetCore-Debugging.md

```diff
@@ -7,11 +7,11 @@ Sometimes it may be necessary to make changes in [`dotnet/aspnetcore`](https://g
 3. Make the desired changes in `dotnet/aspnetcore`.
 4. `./eng/build.cmd -pack`. The `-pack` option causes the creation of NuGet packages.
 5. You should see the generated packages in the `aspnetcore\artifacts\packages\Debug\NonShipping` directory. The packages should end with `x.0.0-dev.nupkg` where `x` is the current .NET version.
-6. Open `aspnetcore-tooling/NuGet.config` and add the local package sources:
+6. Open `razor-tooling/NuGet.config` and add the local package sources:
```

Comment:
This file should probably eventually be changed to `Razor-Compiler-Debugging`, but the compiler repo hasn't set up their docs yet. Filed an issue: https://github.com/dotnet/razor-compiler/issues/11